src/Workspaces/Core/Portable/Simplification/SimplificationOptions.cs

```diff
@@ -45,11 +46,32 @@ public static class SimplificationOptions
         public static Option<bool> AllowSimplificationToBaseType { get; } = new Option<bool>(NonPerLanguageFeatureName, "AllowSimplificationToBaseType", true);
 
         /// <summary>
-        /// This option says if we should simplify away the this. or Me. in member access expression
+        /// This option says if we should simplify away the <see langword="this"/>. or <see langword="Me"/>. in member access expressions.
         /// </summary>
+        [Obsolete]
         public static PerLanguageOption<bool> QualifyMemberAccessWithThisOrMe { get; } = new PerLanguageOption<bool>(PerLanguageFeatureName, "QualifyMemberAccessWithThisOrMe", defaultValue: false);
```

Comment:
I think it should be fully deprecated and no longer used.

@DustinCampbell: I thought about using it as a parent option like you suggested (e.g., setting to `true` or `false` would set/unset all child options), but what would be the expected value if some other extension reads this option?  If all child values are `true`, return `true`; if all child values are `false`, return false, but what about if only some of the child options are `true`?  What do we return then?  We could change it to a `bool?`, but that's still a breaking API change and still doesn't properly convey the information we need.